source/_integrations/googe_drive.markdown

```diff
@@ -0,0 +1,85 @@
+---
+title: Google Drive
+description: Instructions on how to use Google Drive in Home Assistant.
+ha_category:
+  - Utility
+ha_iot_class: Cloud Polling
+ha_release: 2022.10
+ha_config_flow: true
+ha_domain: google_drive
+ha_codeowners:
+  - '@tkdrob'
+ha_integration_type: integration
+---
+
+The Google Drive integration allows you to connect your [Google Drive](https://drive.google.com) to Home Assistant. The integration adds a service to allow you to append rows to a Sheets document. The idea is that you can store data on there for further processing. When you set up a config entry, your drive will have a new sheet called Home Assistant. You can then rename this to whatever you like.
+
+**Note**:
+The integration currently only has access to that one document that is created during setup.
+
+## Prerequisites
+
+You need to configure developer credentials to allow Home Assistant to access your Google Account.
+These credentials are not the same as the ones for [Google Calendar](https://www.home-assistant.io/integrations/google).
+If you have already set up Google Calendar, you can do step 1 and then skip to step 13 on the below instructions.
+
+{% details "Generate Client ID and Client Secret" %}
+
+This section explains how to generate a Client ID and Client Secret on
+[Google Developers Console](https://console.developers.google.com/start/api?id=drive).
+
+1. First go to the [Google Developers Console](https://console.developers.google.com/start/api?id=drive)
+2. The wizard will ask you to choose a project to manage your application. Select a project and click continue.
+3. Verify that your drive API was enabled and click 'Go to credentials'
+4. Navigate to APIs & Services (left sidebar) > [Credentials](https://console.cloud.google.com/apis/credentials)
+5. Click on the field on the left of the screen, **OAuth Consent Screen**.
+6. Select **External** and **Create**.
+7. Set the *App Name* (the name of the application asking for consent) to anything you want e.g.  *Home Assistant*.
```

Comment:
I think there also needs to be a step somewhere that moves the consent screen from *Testing* to *Production* after the consent page is created? Otherwise I think you get logged out every 7 days